src/mainboard/pcengines/apu2/vboot-ro.fmd

```diff
@@ -0,0 +1,19 @@
+FLASH 8M {
+	SI_BIOS@0x0 0x800000 {
+		MISC_RW@0x0 0xa0000 {
+			BOOTORDER(PRESERVE)@0x0 0x1000
+			RW_VPD(PRESERVE)@0x1000 0x4000
+			PSPDIR(PRESERVE)@0x20000 0x80000
+		}
```

Comment:
@piotr-kleins you have to fix up offsets. Look:

PSPDIR ends at 0x20000+0x80000=0xa0000
WP_RO starts at 0x200000
You have a missing space of 0x200000-0xa0000 which must be filled. For example:
```
PSPDIR(PRESERVE)@0x20000 0x80000
RW_UNUSED@0xa0000 0x160000
}
WP_RO@0x200000 0x600000 {
```